pkg/api/types.go

```diff
@@ -65,6 +69,17 @@ type Properties struct {
 	AzProfile               *AzProfile               `json:"azProfile,omitempty"`
 }
 
+// ClusterMetadata represents the metadata of the ACS cluster.
```

Comment:
s/ACS/AKS

Reply:
or, if it's not AKS-specific (i.e., if it also applies to acs-engine built clusters), then just say "Kubernetes cluster" in the comment